### PR TITLE
Add Pinepods

### PR DESCRIPTION
**Pull request by madeofpendletonwool**
Added in Pinepods Podcast Management Platform

Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

**Review by ryangurn — COMMENTED** [review]
LGTM!

File: software/pinepods.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: "Pinepods"
+website_url: "https://www.pinepods.online/"
+source_code_url: "https://github.com/madeofpendletonwool/PinePods"
+description: "PinePods is a Rust based podcast management system that manages podcasts with multi-user support and relies on a central database with clients to connect to it. It's browser based and your podcasts and settings follow you from device to device due to everything being stored on the server. It works on mobile devices and can also sync with a Nextcloud server so you can use external apps Like Antennapod as well!"
```
Comment (Rabenherz112) [inline software/pinepods.yml:4]: > description of what the software does, shorter than 250 characters, sentence case

A possible alternative description below, but feel free to change it however you want to make it get it under 250 characters.

```suggestion
description: "A podcast management system with multi-user support which relies on a central database with clients to connect to it."
```

File: software/pinepods.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+licenses:
+  - GPL-3.0
+platforms:
+  - Docker
+tags:
+  - Media Streaming (Audio Streaming)
+depends_3rdparty: false
```
Comment (Rabenherz112) [inline software/pinepods.yml:11]: This can be omitted, as the default case is false.

```suggestion
```

File: software/pinepods.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+demo_url: "https://try.pinepods.online"
+related_software_url: "https://my.awesome.softwar.e/apps"
```
Comment (Rabenherz112) [inline software/pinepods.yml:13]:
```suggestion
```

**Review by Rabenherz112 — CHANGES_REQUESTED** [review]
On a side note, how do I use the demo? I've set up an account called "Dummy," but I'm struggling to test anything within it, all I seem to see are empty pages?

**Comment by madeofpendletonwool** [comment]
> On a side note, how do I use the demo? I've set up an account called "Dummy," but I'm struggling to test anything within it, all I seem to see are empty pages?

You probably need to search for and add some podcasts. Nothing will show up on the pages other than messages instructing you to add podcasts until you've added a feed of your own. You can search for podcasts with the search bar at the top right.

**Comment by madeofpendletonwool** [comment]
Changes have been made as requested. Thanks for the review!

**Review by Rabenherz112 — APPROVED** [review]

**Review by nodiscc — APPROVED** [review]

**Review by ryangurn — APPROVED** [review]
LGTM!

**Pull request merged**